pkg/volume/util/util.go

```diff
@@ -614,6 +616,22 @@ func FsUserFrom(pod *v1.Pod) *int64 {
 	return fsUser
 }
 
+func HonorFsUser(f *FileProjection, fsUser *int64) {
```

Comment:
I'm not sure what name is appropriate, but this function's name doesn't quite seem to match its effect.

Reply:
I'll rename it to `SetFsUserIfUsernsEnabled`